### ISSUE TITLE
Not work properly with @reach/router

### ISSUE THREAD

**Issue by zcmgyu** [microapps/gatsby-plugin-react-i18next#18]
Presently, I’m so sorry for a great plugin like `gatsby-plugin-react-i18next` it's not working properly with `@reach/router`. I think that the `@reach/router` is a popular library used in Gatsby.

# Reproduction:

In `pages/auth.js`, I’ve a list of routes like this

```
const localizedPath = "/vi/auth"
```

```
<Router basepath={localizedPath}>
    <Redirect from="/" to="sign-in" />
    <SignIn path="sign-in" />
    <SignUp path="sign-up" />
</Router>
```

While the current language is `vi` I enter a URL have path without localized like
https://example.com/auth/sign-up Afterwards, this plugin will automatically append `/vi` into the original path is `/vi/auth`, not the current requested path is `/vi/auth/sign-up`.

Then, @reach/router just do what I arranged is to redirect to `/vi/auth/sign-in` path. So, users couldn't access to sign up page without adding current language code into the path.

# Version

gatsby: 2.24.12
gatsby-plugin-react-i18next: 0.0.24
@reach/router: 1.3.4

**Comment by zcmgyu** [comment]
This is my workaround:

> const newUrl = withPrefix(`/${detected}${originalPath}${queryParams}`);

Instead of the redirect to the original path, I allowed the users to access their requested URL.

> const newUrl = withPrefix(`/${detected}${location.path}${queryParams}`);

Did you agree with this work? If it is OK, I will make a PR.

**Comment by rubenmoya** [comment]
Anything we can do to push this forward? I have the same problem in a project I'm working on

**Comment by jimmyn** [comment]
Fixed in [v0.0.25](https://github.com/microapps/gatsby-plugin-react-i18next/releases/tag/v0.0.25). @zcmgyu thanks for the workaround.
